src/providers/bttv/BttvEmotes.cpp

```diff
@@ -77,6 +77,10 @@ namespace {
 
                 auto id = EmoteId{jsonEmote.value("id").toString()};
                 auto name = EmoteName{jsonEmote.value("code").toString()};
+                auto author = EmoteAuthor{jsonEmote.value("user")
+                                              .toObject()
+                                              .value("name")
+                                              .toString()};
```

Comment:
Is the Author just never part of the BTTV Emote response?
If it's never included, we probably shouldn't try to do any parsing since we don't know the scheme.

Reply:
Channel Emotes and Shared emotes return different JSON objects. Shared Emotes contains the "user" object which contains the "name" of the person who uploaded the emote (what I'm referring to as the author of the emote.)

Here are what the JSON objects look like:

## Shared Emotes 
<img width="453" alt="sharedEmotes" src="https://user-images.githubusercontent.com/19235198/83976283-f0253400-a8ad-11ea-9eb0-23d44503b011.png">

## Channel Emotes
<img width="489" alt="channelEmotes" src="https://user-images.githubusercontent.com/19235198/83976309-25ca1d00-a8ae-11ea-910b-e180b616b487.png">